lang/Polski.json

```diff
@@ -152,9 +152,9 @@
                 "campaigns": "Zobacz wszystkie kampanie na Twitchu"
             },
             "how_it_works": "Jak to działa?",
-            "how_it_works_text": "Co 60 sekund aplikacja wysyła zdarzenie „obejrzana minuta” do kanału, który jest aktualnie oglądany - to wystarczy, aby ukończyć dropienie. Zauważ, że w ten sposób możesz całkowicie pominąć konieczność pobierania rzeczywistego strumienia wideo i dźwięku. Do utrzymania statusu ONLINE lub OFFLINE kanałów, używane jest połączenie WebSocket, które odbiera zdarzenia o zmianie statusu kanałów, oraz aktualizuje aktualną ilości widzów.",
+            "how_it_works_text": "Co minutę aplikacja wysyła zdarzenie „obejrzana minuta” do kanału, który jest aktualnie oglądany - to wystarczy, aby ukończyć dropienie. Zauważ, że w ten sposób możesz całkowicie pominąć konieczność pobierania rzeczywistego strumienia wideo i dźwięku. Do utrzymania statusu ONLINE lub OFFLINE kanałów używane jest połączenie WebSocket, które odbiera zdarzenia o zmianie statusu kanałów oraz aktualizuje aktualną ilość widzów.",
             "getting_started": "Pierwsze kroki",
-            "getting_started_text": "• Zaloguj się do aplikacji.\n• Upewnij się, że twoje konto Twitch jest połączone ze wszystkimi kampaniami, które chcesz dropić.\n• Jeśli interesuje cię dropienie wszystkiego, co możliwe to odznacz „Tylko priorytetowe” i naciśnij na „Przeładuj”.\n• Jeśli chcesz otrzymać dropy z konkretnej gry, użyj listy „Priorytety”, aby wybrać uporządkowaną listę gier, których dropy chcesz otrzymać. Gry z górnej części listy będą miały większy priorytet dropienia, niż te niżej na liście.\n• Zaznacz opcję „Tylko priorytetowe”, aby wykluczyć z dropienia gry, które nie są na liście priorytetowej.\n• Użyj listy „Wykluczone”, aby wskazać gry, których kampanie nigdy nie powinny być dropione.\n• Zmiana zawartości list lub opcji „Tylko priorytetowe” wymaga naciśnięcia na „Przeładuj”, aby zastosować zmiany."
+            "getting_started_text": "1. Zaloguj się do aplikacji.\n2. Upewnij się, że twoje konto Twitch jest połączone ze wszystkimi kampaniami, z których chcesz wydropić przedmioty.\n3. Odznacz opcję „Tylko priorytetowe” i kliknij „Przeładuj”, jeśli chcesz rozpocząć dropienie wszystkich przedmiotów z aktywnych kampanii.\n4. Użyj listy „Priorytety”, aby wybrać konkretną grę lub uporządkować listę gier, z których chcesz wydropić przedmioty. Gry z górnej części listy będą miały większy priorytet dropienia niż te niżej na liście.\n5. Zaznacz opcję „Tylko priorytetowe”, aby opcjonalnie wykluczyć z dropienia gry, które nie są na liście priorytetowej - to już zależy od ciebie.\n6. Użyj listy „Wykluczone”, aby wskazać aplikacji gry, z których przedmioty nigdy nie powinny być dropione.\n7. Zmiana zawartości list lub opcji „Tylko priorytetowe” wymaga kliknięcia „Przeładuj”, aby aplikacja mogła zastosować wprowadzone zmiany."
```

Comment:
This change introduces a new word: "przedmioty", and the meaning of them "coming from" a campaign. A direct translation of "przedmiot" is "item", not a "drop". The rest of the translation file refers to drops simply as a "drop", and "getting drops" is translated as "dropić" or "dropienie", which can remain valid under the circumstances that a literal translation of "zrzut" doesn't really fit in here . Using this new "przedmioty" word is a yet another inconsistency, and the simplest way to deal with it would be to go back to referring to it drops as "drop" only. Please note that each drop can have multiple "rewards" to it, and that's where the "items" translation could come in handy, at some point later on in the development, if a label to it would be ever needed.

Other than that, the priority list is for "a list of games", not "a particular game". Point #​5 doesn't need the "optionally" word there, that's like saying "an option makes things optional" which is redundant, similarly to "this depends entirely on you" at the end there - that's what all these options are for in the first place, so lets keep it short and to the point.

My suggestion implements all of the above. Because you've restructured #​2 entirely, #​2 and #​4 ended with the same sentence ending now, which isn't ideal. My suggestion is to go back with #​2 to how it was before (and finally put the comma at the right place). Everything else is fine.

```suggestion
            "getting_started_text": "1. Zaloguj się do aplikacji.\n2. Jeśli interesuje cię dropienie wszystkiego co możliwe, to odznacz „Tylko priorytetowe” i naciśnij na „Przeładuj”\n3. Odznacz opcję „Tylko priorytetowe” i kliknij „Przeładuj”, jeśli chcesz rozpocząć dropienie z wszystkich aktywnych kampanii.\n4. Użyj listy „Priorytety”, aby wybrać uporządkowaną listę gier, które chcesz dropić. Gry z górnej części listy będą miały większy priorytet dropienia niż te niżej na liście.\n5. Zaznacz opcję „Tylko priorytetowe”, aby wykluczyć z dropienia gry, które nie są na liście priorytetowej.\n6. Użyj listy „Wykluczone”, aby wskazać aplikacji gry, z których przedmioty nigdy nie powinny być dropione.\n7. Zmiana zawartości list lub opcji „Tylko priorytetowe” wymaga kliknięcia „Przeładuj”, aby aplikacja mogła zastosować wprowadzone zmiany."
```

Reply:
I simply translated the English description. 

Point 2: Missing that linking is required in your suggestion `(Ensure your Twitch account is linked...)`. 

Point 4: `"konkretną grę" (if the user wants to add only 1 game to the list)` refers to the English `specific game (If you want to mine specific game first...)` and I added "ordered list" if the user would like to make a list of games to drop. This can be shortened to an ordered list - as you suggested. 

Point 2: What do you think about restoring the missing `"Ensure your Twitch account is linked..."`.

Suggestion - Point 2:
`Upewnij się, że twoje konto Twitch jest połączone ze wszystkimi kampaniami. Jeśli interesuje cię dropienie wszystkiego co możliwe, to odznacz „Tylko priorytetowe” i naciśnij na „Przeładuj”.`

Reply:
Hmm, actually, that's my bad for mistaking point #​2 and #​3 there. Not sure why this has happened.

Regardless, if #​2 is changed to avoid using the word "przedmioty", it becomes:

> Upewnij się, że twoje konto Twitch jest połączone ze wszystkimi kampaniami, z których chcesz dropić.

This makes it end with almost the same words as the first sentence of point #4:

> Użyj listy „Priorytety”, aby wybrać uporządkowaną listę gier, które chcesz dropić.

Regarding point #4 too, what you've said about "konkretną grę" does make some sense, however it has to be used appropriately. A user can be "interested in a particular/specific game", but then they have to add it to "a list of games". Even with one game added, it's still a list, thus you can refer to the list only as "specific games", not "a specific game or a list". Now, regarding the translation itself, the Polish language structure makes it so that there's a reversal of the word ordering there, and thus you first have to refer to setting up the list of games, and then you specify that those games are the ones you're interested in, hence the sentence I proposed does make sense. "Konkretne gry" is already included in the "które chcesz dropić" part of the sentence, so there's no need to include it anywhere else.

To resolve the similar wording of point #4, you can go back to the previous "których dropy chcesz otrzymać" translation. It's probably not the best, but at least avoids the problem.

```suggestion
            "getting_started_text": "1. Zaloguj się do aplikacji.\n2. Upewnij się, że twoje konto Twitch jest połączone ze wszystkimi kampaniami, z których chcesz dropić.\n3. Odznacz opcję „Tylko priorytetowe” i kliknij „Przeładuj”, jeśli chcesz rozpocząć dropienie z wszystkich aktywnych kampanii.\n4. Użyj listy „Priorytety”, aby wybrać uporządkowaną listę gier, których dropy chcesz otrzymać. Gry z górnej części listy będą miały większy priorytet dropienia niż te niżej na liście.\n5. Zaznacz opcję „Tylko priorytetowe”, aby wykluczyć z dropienia gry, które nie są na liście priorytetowej.\n6. Użyj listy „Wykluczone”, aby wskazać aplikacji gry, z których przedmioty nigdy nie powinny być dropione.\n7. Zmiana zawartości list lub opcji „Tylko priorytetowe” wymaga kliknięcia „Przeładuj”, aby aplikacja mogła zastosować wprowadzone zmiany."
```

Reply:
```suggestion
            "getting_started_text": "1. Zaloguj się do aplikacji.\n2. Upewnij się, że twoje konto Twitch jest połączone ze wszystkimi kampaniami, z których chcesz dropić.\n3. Odznacz opcję „Tylko priorytetowe” i kliknij „Przeładuj”, jeśli chcesz rozpocząć dropienie ze wszystkich aktywnych kampanii.\n4. Użyj listy „Priorytety”, aby wybrać uporządkowaną listę gier, z których chcesz otrzymać dropy. Gry z górnej części listy będą miały większy priorytet dropienia niż te niżej na liście.\n5. Zaznacz opcję „Tylko priorytetowe”, aby wykluczyć z dropienia gry, które nie są na liście priorytetowej.\n6. Użyj listy „Wykluczone”, aby wskazać aplikacji gry, z których przedmioty nigdy nie powinny być dropione.\n7. Zmiana zawartości list lub opcji „Tylko priorytetowe” wymaga kliknięcia „Przeładuj”, aby aplikacja mogła zastosować wprowadzone zmiany."
```